ganga/GangaCore/Runtime/bootstrap.py

```diff
@@ -1220,6 +1218,7 @@ def launch_IPython(local_ns, error_handler, ganga_prompt):
         import IPython
 
         # Based on examples/Embedding/embed_class_long.py from the IPython source tree
+        from IPython.core.displayhook import DisplayHook
```

Comment:
Looking at the diff of this patch, unless I missed it, DisplayHook is never used so does it need to be imported?

Reply:
Ah yes - this was my trying to alter the display hook.